src/NuGetPackageVerifier/Rules/PackageVersionMatchesAssemblyVersionRule.cs

```diff
@@ -6,40 +6,33 @@
 using System.Linq;
 using System.Reflection;
 using Mono.Cecil;
+using Mono.Collections.Generic;
+using NuGet.Packaging;
 using NuGet.Versioning;
 
 namespace NuGetPackageVerifier.Rules
 {
     public class PackageVersionMatchesAssemblyVersionRule : AssemblyHasAttributeRuleBase
```

Comment:
Better to have this implement `IPackageVerifierRule` instead of using `AssemblyHasAttributeRuleBase` and copy the boilerplate code into this class.